lib/cli.js

```diff
@@ -40,11 +40,22 @@ function parse(path) {
 }
 
 function processFile(filePath) {
-  let source = fs.readFileSync(filePath, "utf8");
-  let formatted = formatter.formatJson(source);
-  let fileWithErrors = showDiff(program.quiet, source, formatted, filePath);
+  let formatted, source, fileWithErrors;
```

Comment:
You're only using those variables inside the try block, and you're not mutating them. So you can just define'm as `const`s there